tests/vmi_lifecycle_test.go

```diff
@@ -129,17 +129,13 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 		})
 
 		It("[test_id:1620]should start it", func() {
-			vmi, err := virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(vmi)).Create(context.Background(), vmi)
-			Expect(err).ToNot(HaveOccurred(), "Create VMI successfully")
-			libwait.WaitForSuccessfulVMIStart(vmi)
+			tests.RunVMIAndExpectLaunch(vmi, 30)
```

Comment:
I really do not like the unification done here.
But I know it was a review request, so I won't start insisting on this now.

For the record:
- I think the existing is good. It should be clear in the test that a VMI is created and then waited for. Merging into one function may be "clean" in the sense of less code, but it hides what happens.
- The function is located in a place we want to get rid of, using it now, is not that nice.

Reply:
Would you prefer it split out into its own function here?

Something like `createAndStartVMI` as discussed previously?

Reply:
Nope, I argue that there is no need to abstract the creation and the wait further.

Reply:
Are you referring to just this test or every instance of this abstraction?

Reply:
I refer to the changes done in this PR.
Again, I will not insist on this because you got a different comment to add it and this may be a matter of disagreement. And I do not want to block this change on it.